analyzers/its/sources/Net7/Net7/features/AvoidDateTimeNowForBenchmarking.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Net7.features
+{
+    internal class AvoidDateTimeNowForBenchmarking
+    {
+        public void S6561()
+        {
+            var start = DateTime.Now;
+            MethodToBeBenchmarked();
+            Console.WriteLine($"{(DateTime.Now - start).TotalMilliseconds} ms");
+        }
+
+        void MethodToBeBenchmarked() { }
+    }
```

Comment:
Since I'm not raising any issue on ITs I added this example to verify the whole process. As soon as we are able to run ITs again I'll update the expected issues accordingly.

Reply:
Please add the case in `ManuallyAddedNoncompliantIssues.CS`. The NET projects are for syntax/ types specific to that framework.